src/libcore/fmt/float.rs

```diff
@@ -22,6 +22,9 @@ fn float_to_decimal_common_exact<T>(fmt: &mut Formatter, num: &T,
     unsafe {
         let mut buf = MaybeUninit::<[u8; 1024]>::uninitialized(); // enough for f32 and f64
         let mut parts = MaybeUninit::<[flt2dec::Part; 4]>::uninitialized();
+        // FIXME: Technically, this is calling `get_mut` on an uninitialized
```

Comment:
Maybe file an issue about this specifically, and mention that issue number in code comments for all known occurences? That way when the question is resolved we can grep for the issue number to find places to fix (or to remove outdated FIXME comments).

Reply:
Wouldn't this usually be part of the discussion in the [tracking issue](https://github.com/rust-lang/rust/issues/53491)?

Reply:
I agree. Is like this okay? This covers all `get_mut` uses that got added as part of my PR series. I grepped the codebase for other uses of `MaybeUninit` and found none.